### PR TITLE
Translate id with Any Child Of condition

### PR DESCRIPTION
**Pull request by GeorgeBotsev**
When using Any Child Of condition for Elementor templates, make sure that we translate the ID

https://onthegosystems.myjetbrains.com/youtrack/issue/wpmlcore-7559

### CODE REVIEW

File: tests/phpunit/tests/test-wpml-elementor-translate-ids.php [hunk 1]
```diff
@@ -373,4 +373,30 @@ public function dp_global_widget() {
...
 		);
 	}
+
+	/**
+	 * @test
+	 * @group wpmlcore-7559
+	 */
+	public function it_should_translate_location_condition_any_child_of() {
+		$sub_id           = mt_rand( 1, 10 );
+		$translate_id     = mt_rand( 11, 20 );
```
Comment (strategio) [inline tests/phpunit/tests/test-wpml-elementor-translate-ids.php]: @GeorgeBotsev, let's replace random numbers with hardcoded ones:

```php
		$sub_id           = 123;
		$translate_id     = 456;
```

File: tests/phpunit/tests/test-wpml-elementor-translate-ids.php [hunk 2]
```diff
@@ -373,4 +373,30 @@ public function dp_global_widget() {
...
+		$post_type        = 'page';
+		$parsed_conditions = array( 'sub_name' => 'any_child_of' );
```
Comment (strategio) [inline tests/phpunit/tests/test-wpml-elementor-translate-ids.php]:
```suggestion
		$parsed_conditions = [ 'sub_name' => 'any_child_of' ];
```

File: tests/phpunit/tests/test-wpml-elementor-translate-ids.php [hunk 3]
```diff
@@ -373,4 +373,30 @@ public function dp_global_widget() {
...
+
+		\WP_Mock::userFunction( 'get_post_type',
+			array(
```
Comment (strategio) [inline tests/phpunit/tests/test-wpml-elementor-translate-ids.php]: Please use short syntax here as well.

**Review by strategio — DISMISSED** [review]
It looks good @GeorgeBotsev. I just have some minor comments for the test :)

**Review by dgwatkins — DISMISSED** [review]
:+1: looks good

File: tests/phpunit/tests/test-wpml-elementor-translate-ids.php (hunk 1 above)
Comment (GeorgeBotsev) [reply tests/phpunit/tests/test-wpml-elementor-translate-ids.php]: Fixed

File: tests/phpunit/tests/test-wpml-elementor-translate-ids.php (hunk 2 above)
Comment (GeorgeBotsev) [reply tests/phpunit/tests/test-wpml-elementor-translate-ids.php]: Fixed

File: tests/phpunit/tests/test-wpml-elementor-translate-ids.php (hunk 3 above)
Comment (GeorgeBotsev) [reply tests/phpunit/tests/test-wpml-elementor-translate-ids.php]: Implemented as suggested.

**Review by dgwatkins — DISMISSED** [review]

**Pull request merged**